include/sys/arc_impl.h

```diff
@@ -876,6 +853,96 @@ typedef struct arc_stats {
 	kstat_named_t arcstat_abd_chunk_waste_size;
 } arc_stats_t;
 
+typedef struct arc_wmsums {
```

Comment:
The naming is awkward here. Traditionally, we prefix the element names based on the struct type name.
            - I get that this is a mapping to the arc_stats_t struct above
            - but maybe we could consider a `sum_` prefix (or `arcsum_ `)?

Reply:
It is impossible without previous.